### PR TITLE
Release core-react & tokens

### LINKED ISSUE #1122: Release new version of libraries
Publish new releases that are under draft: https://github.com/equinor/design-system/releases

Do it manually this time and setup github actions after this issue is done

- [ ] Publish Storybook
- [ ] Publish npm packages

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1122 
Added changelog & bumped version

Release logs: https://github.com/equinor/design-system/releases

### CODE REVIEW

File: libraries/core-react/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,35 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.5] - 2021-02-22
+
+### Added ✨
+
+- Progress indicators ([#1090](https://github.com/equinor/design-system/issues/1090), [#1098](https://github.com/equinor/design-system/issues/1098))
+  - `Circular`, `Star` and `Dots` now have a `size` property for adjusting size.
+  - `Circular` and `Dots` now have a `color` property following the same color scheme as our other components.
+  - Added compound component support. You can now find the progress indicators as `Progress.Dots`, `Progress.Star` etc..
```
Comment (vnys) [inline libraries/core-react/CHANGELOG.md:15]:
```suggestion
  - Refactored as compound components. You can now find the progress indicators as `Progress.Dots`, `Progress.Star` etc..
```

**Review by vnys — CHANGES_REQUESTED** [review]

File: libraries/core-react/CHANGELOG.md (hunk 1 above)
Comment (vnys) [reply libraries/core-react/CHANGELOG.md:15]: Since we’re not actually _supporting_ compound components, but offering them…

Comment (mimarz) [reply libraries/core-react/CHANGELOG.md:15]: @vnys We still have the individual sub-components being exported as `LinearProgress`, `CircularProgress` etc, so maybe also we should mention that they will be removed in the future

Comment (vnys) [reply libraries/core-react/CHANGELOG.md]: Yes, I agree

Comment (mimarz) [reply libraries/core-react/CHANGELOG.md]: Updated: 

>     - The individual `LinearProgress`, `CircularProgress`, `StarProgress` & `DotProgress` will be removed at a later time

**Review by vnys — APPROVED** [review]
👍🏻

**Pull request merged**